R/dsl.R

```diff
@@ -156,15 +156,12 @@ do_package_checks <- function(...,
 #' @inheritParams step_setup_push_deploy
 #' @inheritParams step_do_push_deploy
 #' @param build_only Build the pkgdown site but do not deploy it.
```

Comment:
This now needs a comment that the deployment never happens if we can't push. Maybe we should move the `ci_can_push()` check into the corresponding steps, and link explicitly to these steps?

Reply:
Good idea, makes sense.